.tx/config

```diff
@@ -0,0 +1,20 @@
+[main]
+host = https://www.transifex.com
+lang_map = pt_BR: pt-BR, pt_PT: pt-PT, zh_CN: zh-Hans, zh_TW: zh-Hant
+minimum_perc = 80
```

Comment:
If #106 lands first, we’ll want to add Navigation.storyboard to the resources and give it a `minimum_perc` much lower than 80%, since most of the strings are “DO NOT TRANSLATE”.